<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroCliente</class>
 <widget class="QDialog" name="CadastroCliente">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Clientes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_14">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Novo cadastro</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="4" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_9">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Mais informações</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="4" column="0">
                <widget class="QLabel" name="label_41">
                 <property name="text">
                  <string>Vendedor</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_44">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Segmento de mercado</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="2">
                <widget class="QCheckBox" name="checkBox_4">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>limpar</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_43">
                 <property name="text">
                  <string>Vinculado ao cliente</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="3">
                <widget class="ComboBox" name="comboBoxCliente"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_42">
                 <property name="text">
                  <string>Profissional</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0" colspan="3">
                <widget class="QLineEdit" name="lineEditSegmento">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="5" column="0" colspan="3">
                <widget class="ComboBox" name="comboBoxVendedor"/>
               </item>
               <item row="3" column="0" colspan="3">
                <widget class="ComboBox" name="comboBoxProfissional"/>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_40">
                 <property name="text">
                  <string>Clientes vinculados ao cliente</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="textEditClientes">
                 <property name="textInteractionFlags">
                  <set>Qt::NoTextInteraction</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBoxDadosPessoais">
            <property name="title">
             <string>Dados Pessoais</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="4">
              <widget class="QLineEdit" name="lineEditRG">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEditApelido">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditCPF">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelCPF">
               <property name="text">
                <string>CPF</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_1">
               <property name="text">
                <string>RG</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="labelApelido">
               <property name="text">
                <string>Apelido</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelNome">
               <property name="text">
                <string>Nome</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="lineEditNome">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>labelCPF</zorder>
            <zorder>lineEditCPF</zorder>
            <zorder>labelApelido</zorder>
            <zorder>lineEditApelido</zorder>
            <zorder>label_1</zorder>
            <zorder>lineEditRG</zorder>
            <zorder>labelNome</zorder>
            <zorder>lineEditNome</zorder>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Tipo</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="radioButtonCliente">
               <property name="text">
                <string>Cliente</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonForn">
               <property name="text">
                <string>Fornecedor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonAmbos">
               <property name="text">
                <string>Ambos</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBoxPJuridica">
            <property name="title">
             <string>Info. Pessoa Jurídica</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="1" column="0">
              <widget class="QWidget" name="widgetPJ" native="true">
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_38">
                  <property name="text">
                   <string>Razão Social</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_54">
                  <property name="text">
                   <string>Nome Fantasia</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_39">
                  <property name="text">
                   <string>CNPJ</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="label_53">
                  <property name="text">
                   <string>Insc. Estadual</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEditRazaoSocial">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 127);</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEditNomeFantasia">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="lineEditCNPJ">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>135</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 127);</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLineEdit" name="lineEditInscEstadual">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>135</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 127);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBoxContatos">
            <property name="title">
             <string>Contatos</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="1">
              <widget class="LineEditTel" name="lineEditTel_Cel">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>ID Nextel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Nextel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>E-mail</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Telefone Principal</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Telefone celular</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Telefone comercial</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="LineEditTel" name="lineEditTel_Res">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="LineEditTel" name="lineEditNextel">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="LineEditTel" name="lineEditTel_Com">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLineEdit" name="lineEditIdNextel">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="6">
              <widget class="QLineEdit" name="lineEditEmail">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhEmailCharactersOnly</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Endereços</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Endereço do Faturamento (Principal)</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0" colspan="2">
              <widget class="WidgetEndereco" name="widgetEnd_1" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>70</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBoxCobranca">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Endereço de Cobrança</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="WidgetEndereco" name="widgetEnd_2" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBoxEntrega">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Endereço de Entrega</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0">
              <widget class="WidgetEndereco" name="widgetEnd_3" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
         <zorder>groupBox_3</zorder>
         <zorder>groupBoxEntrega</zorder>
         <zorder>groupBoxCobranca</zorder>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonCadastrar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cadastrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAtualizar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Atualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBuscar">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/search.png</normaloff>:/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNovoCad">
       <property name="text">
        <string>Novo Cadastro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditTel</class>
   <extends>QLineEdit</extends>
   <header>lineedittel.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetEndereco</class>
   <extends>QWidget</extends>
   <header>widgetendereco.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditRazaoSocial</tabstop>
  <tabstop>lineEditNomeFantasia</tabstop>
  <tabstop>lineEditCNPJ</tabstop>
  <tabstop>lineEditInscEstadual</tabstop>
  <tabstop>radioButtonCliente</tabstop>
  <tabstop>radioButtonForn</tabstop>
  <tabstop>radioButtonAmbos</tabstop>
  <tabstop>lineEditNome</tabstop>
  <tabstop>lineEditCPF</tabstop>
  <tabstop>lineEditApelido</tabstop>
  <tabstop>lineEditRG</tabstop>
  <tabstop>lineEditTel_Res</tabstop>
  <tabstop>lineEditTel_Cel</tabstop>
  <tabstop>lineEditTel_Com</tabstop>
  <tabstop>lineEditIdNextel</tabstop>
  <tabstop>lineEditNextel</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>comboBoxCliente</tabstop>
  <tabstop>comboBoxProfissional</tabstop>
  <tabstop>comboBoxVendedor</tabstop>
  <tabstop>lineEditSegmento</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>textEditClientes</tabstop>
  <tabstop>pushButtonCadastrar</tabstop>
  <tabstop>pushButtonAtualizar</tabstop>
  <tabstop>pushButtonBuscar</tabstop>
  <tabstop>pushButtonNovoCad</tabstop>
  <tabstop>pushButtonRemover</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>groupBoxEntrega</tabstop>
  <tabstop>groupBoxCobranca</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
